src/simulation/LoadGenerator.cpp

```diff
@@ -562,6 +655,86 @@ LoadGenerator::generateLoad(GeneratedLoadConfig cfg)
     scheduleLoadGeneration(cfg);
 }
 
+uint32_t
+LoadGenerator::submitSorobanPrepareInvokeTX(uint32_t nAccounts,
```

Comment:
I'm not sure this works as we don't seem to be handling tx failure modes here: any transaction successfully submitted to the tx queue via loadgen isn't guaranteed to actually make it to block application. Even at block application, it may still fail. From what I see, if this happens loadgen will just get stuck. In other modes, we implement a timeout to get it unstuck (e.g. `waitTillComplete`). It seems the submission pattern that would handle this properly is something like: submit upload wasms and wait until they all show up in DB (timeout and fail if they don't), submit deploy instances, and wait until they are reflected in the DB (with the same timeout strategy), and only then return success in this setup mode